indy_client/client/wallet/wallet.py

```diff
@@ -1,33 +1,27 @@
 import datetime
```

Comment:
Why can't we just remove this class? Where is it used?

Reply:
This class is used in plenum.server.node.Node#wallet. We use it for signing update messages in indy_node.server.node.Node#notify_upgrade_start

Reply:
Removed